app/(main)/login/page.tsx

```diff
@@ -36,17 +47,19 @@ type LoginUserSchemaType = z.infer<typeof LoginUserSchema>;
 
 /**
  * Renders the login page.
- * @returns {JSX.Element} The rendered login page.
+ * @returns {React.JSX.Element} The rendered login page.
  */
-const Login = (): JSX.Element => {
+const Login = (): React.JSX.Element => {
   const router = useRouter();
   const { login, isSignedIn, getUser } = useAuthContext();
+  const [isLoading, setIsLoading] = useState(false);
 
   useEffect(() => {
     if (isSignedIn) {
       getUser();
       router.push('/league/all');
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
No need to add this. It's just a warning.

Reply:
Removed